_data/tasks.yml

```diff
@@ -23,4 +23,7 @@ toc:
   section:
   - title: Assigning Pods to Nodes
     path: /docs/tasks/administer-cluster/assign-pods-nodes/
-
```

Comment:
Updating it as we go is going to cause merge conflicts, but updating it at the end is harder to coordinate. I think we should update it as we go and have the individual authors resolve the merge conflicts; it should just take a second and at least that way everything gets included.